lib/Sema/CSStep.h

```diff
@@ -975,8 +976,8 @@ class ConjunctionStep : public BindingStep<ConjunctionElementProducer> {
       CS.Timer.emplace(anchor, CS, remainingTime);
     }
     
-    if (CS.isDebugMode())
-      getDebugLogger() << ")\n";
+//    if (CS.isDebugMode())
```

Comment:
Should be commented?

Reply:
It should be removed, thanks, @LucianoPAlmeida!